snuba/datasets/schemas/__init__.py

```diff
@@ -107,6 +109,8 @@ def get_column_differences(self, expected_columns: Mapping[str, str]) -> List[st
                 continue
 
             expected_type = self.get_columns()[column_name].type.for_schema()
+
+            column_type = column.for_schema()
             if column_type != expected_type:
```

Comment:
Any reason to compare the string representation in stead of the Type object now that we deal with ColumnType classes around ? I think we should turn this into a ColumnType comparison if it is possible.

Reply:
Not really, done